src/server/mod.rs

```diff
@@ -51,13 +53,13 @@ where
             let app = App::new()
                 .app_data(Data::new(runtime_context.clone()))
                 .wrap(Logger::default())
-                .route("/", web::get().to(index))
+                .route("/", web::get().to(index::<Context, KeyManager>))
                 .route("/tx/{tx_id}", web::get().to(get_tx::<Context>))
                 .service(
                     web::scope("/cosigner")
                         .route("/sign", web::post().to(sign_route::<Context, KeyManager>)),
                 )
-                .service(web::scope("/idle").route("/", web::get().to(index)));
+                .service(web::scope("/idle").route("/", web::get().to(index::<Context, KeyManager>)));
```

Comment:
What is purpose of this? Where would it be used?

Reply:
Checkout: 
https://node1.bundlr.network/
https://node1.bundlr.network/public

It comes with some useful information and stats for monitoring and integrations

Reply:
The goal here is to add similar information to the "/" (index) and "/status" routes

Reply:
Yeah, but why two different entrypoints with same data? Or is the purpose just to have these entrypoints in place and then later provide different data?

Reply:
Yes, later provide different data

Reply:
Oh, actually this was not added. It was just changed to match signature of `index`. My bad.